BedrockServer.cpp

```diff
@@ -772,17 +772,23 @@ void BedrockServer::runCommand(unique_ptr<BedrockCommand>&& _command, bool isBlo
     // our state right before we commit.
     SQLiteNodeState state = _replicationState.load();
 
-    // If we're following, we will automatically escalate any command that's not already complete (complete
-    // commands are likely already returned from leader with legacy escalation) and is marked as
-    // `escalateImmediately` (which lets them skip the queue, which is particularly useful if they're waiting
-    // for a previous commit to be delivered to this follower), OR if we're on a different version from leader.
-    if (state == SQLiteNodeState::FOLLOWING && (_version != _leaderVersion.load() || command->escalateImmediately) && !command->complete) {
+    // If we're following, we will automatically escalate any command that's:
+    // 1. Not already complete (complete commands are likely already returned from leader with legacy escalation) 
```

Comment:
>complete commands are likely already returned from leader with legacy escalation

I don't understand what this means, what is "legacy escalation"?

Reply:
I don't know too, that was the old comment that was used for leader escalations only. I thought that removing that piece of information would probably be bad for someone in the future

Reply:
@tylerkaraszewski do you know what it means?

Reply:
bump @tylerkaraszewski

Reply:
I don't remember. I think probably it is there to support both escalation over http, and also the old-style escalation on the sync thread. We could potentially remove it but that makes me wonder if we'd still need he check for `complete` here. Do complete commands ever end up here anymore?

Reply:
Would it be good to add a log to see if we do?